### PR TITLE
Muscled Veins experimental surgery

### PR DESCRIPTION
**Pull request by XDTM**
## About The Pull Request

Added a new surgery, Muscled Veins, as the contrast to Vein Threading: it makes the patient able to survive without a heart.

Note: Part of #44440; <s>needs another change for it to compile by itself, which i will make if the other PR containing it is rejected.</s>

## Changelog
:cl: XDTM
add: Added a new advanced surgery, Muscled Veins: it makes the patient able to survive without a heart.
/:cl:

### CODE REVIEW

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -95,15 +95,15 @@
 	description = "When simple medicine doesn't cut it."
 	prereq_ids = list("imp_wt_surgery")
 	design_ids = list("surgery_lobotomy", "surgery_heal_brute_upgrade_femto","surgery_heal_burn_upgrade_femto","surgery_heal_combo","surgery_exp_dissection")
-	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 1500)
```
Comment (nemvar) [inline code/modules/research/techweb/all_nodes.dm]: Is this intentional? It isn't mentioned in the PR. Not that I have anything against it.

**Comment by HomocidlePancakes** [comment]
Cyberheat viable? Wow

File: code/modules/surgery/advanced/experimental_dissection.dm [hunk 2]
```diff
@@ -26,7 +26,9 @@
 	time = 125
 
 /datum/surgery_step/dissection/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
-	user.visible_message("[user] starts dissecting [target].", "<span class='notice'>You start dissecting [target].</span>")
+	display_results(user, target, "<span class='notice'>You start dissecting [target].</span>",
+		"[user] starts dissecting [target].",
+		"[user] begins to make an incision in [target]'s [parse_zone(target_zone)].")
```
Comment (nemvar) [inline code/modules/surgery/advanced/experimental_dissection.dm]: Is this supposed to be part of this PR?

File: code/modules/research/techweb/all_nodes.dm (hunk 1 above)
Comment (XDTM) [reply code/modules/research/techweb/all_nodes.dm]: i forgot that was in the same file, it was part of the big PR

File: code/modules/surgery/advanced/experimental_dissection.dm (hunk 2 above)
Comment (XDTM) [reply code/modules/surgery/advanced/experimental_dissection.dm]: Gah, i forgot i edited this as well

**Comment by XDTM** [comment]
Now that the stealth surgery PR is merged this compiles.

**Review by ExcessiveUseOfCobblestone — COMMENTED** [review]
This really should make you bleed faster or something since it does away with an organ.

If we ever want organ damage to be on all organs this will just make it even more stronger.

File: code/modules/research/techweb/all_nodes.dm [hunk 3]
```diff
@@ -95,15 +95,15 @@
 	description = "When simple medicine doesn't cut it."
 	prereq_ids = list("imp_wt_surgery")
 	design_ids = list("surgery_lobotomy", "surgery_heal_brute_upgrade_femto","surgery_heal_burn_upgrade_femto","surgery_heal_combo","surgery_exp_dissection")
-	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 1500)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
```
[the remark this answers is not in the document]
Comment (XDTM) [reply code/modules/research/techweb/all_nodes.dm]: dunno how that got included, i'll fix it

**Comment by XDTM** [comment]
Consider that the alternative to this is a surgery that makes you 90% resistant to bleeding. I could add drawbacks to both surgeries (and i probably will if they end up being too prominent), but for now i think it's fine as is, with the price for the improvement being time and being vulnerable during surgery.

**Comment by ExcessiveUseOfCobblestone** [comment]
kk im down. If it gets icky one of us will PR a revert anyways.

**Pull request merged**